src/components/Map/__tests__/Map.spec.tsx

```diff
@@ -23,84 +26,131 @@ describe('<Map/> that renders a Map with markers', () => {
+        render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
     });
+
     it('should remove default highlight when markers added', () => {
-        const wrapper = mount(<Map defaultHighlight={NL_PATHS} />);
+        const view = render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
 
-        wrapper.setProps({ markers: [pointMarker, regionMarker] });
+        view.rerender(
+            <Map
+                defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject}
+                markers={[pointMarker, regionMarker]}
+            />
+        );
         // will be called as many times as the length of the returned array from addGeoJsonMock
         expect(removeDataMock).toHaveBeenCalledTimes(1);
     });
+
     it('should render with markers', () => {
-        const wrapper = mount(<Map markers={[pointMarker, regionMarker]} />);
-        expect(wrapper.find('Marker')).toHaveLength(2);
-        expect(wrapper.find('Circle')).toHaveLength(1);
+        const view = render(<Map markers={[pointMarker, regionMarker]} />);
+
+        expect(view.container).toMatchSnapshot();
     });
+
     it('should fit map to markers', () => {
-        mount(<Map markers={[pointMarker, regionMarker]} />);
+        render(<Map markers={[pointMarker, regionMarker]} />);
         // call fitBounds for each marker
         expect(fitBoundsMock).toHaveBeenCalledTimes(1);
     });
+
     it('should fit map when new marker is added', () => {
-        const wrapper = mount(<Map markers={[pointMarker]} />);
+        const view = render(<Map markers={[pointMarker]} />);
+
         expect(fitBoundsMock).toHaveBeenCalledTimes(1);
-        wrapper.setProps({ markers: [pointMarker, regionMarker] });
+
+        view.rerender(
+            <Map
+                defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject}
+                markers={[pointMarker, regionMarker]}
+            />
+        );
+
         // call count = original call + again for each marker + 1 more because markers are updated with a small delay
         expect(fitBoundsMock).toHaveBeenCalledTimes(3);
     });
+
     it('should fit to default center and zoom if markers removed', () => {
-        const wrapper = mount(<Map markers={[pointMarker, regionMarker]} />);
-        wrapper.setProps({ markers: [] });
+        const view = render(<Map markers={[pointMarker, regionMarker]} />);
+        view.rerender(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} markers={[]} />);
+
         expect(setZoomMock).toHaveBeenCalledTimes(1);
         expect(setCenterMock).toHaveBeenCalledTimes(1);
     });
+
     it('should add highlighted areas when geoJson objects are passed as markers', () => {
-        mount(<Map markers={[FR_FRIESLAND]} />);
+        render(<Map markers={[FR_FRIESLAND as FeatureCollection]} />);
```

Comment:
why are you using a different type then what is in the Prop definition in the component?